python/cuml/cuml/internals/logger.pyx

```diff
@@ -195,10 +138,16 @@ def set_pattern(pattern):
         pattern for a code section, as described in the example section above.
     """
     IF GPUBUILD == 1:
-        cdef string prev = Logger.get().getPattern()
-        context_object = PatternSetter(prev.decode("UTF-8"))
+        # TODO: We probably can't implement this exact API because you can't
```

Comment:
Can you create a github issue to track this?

Reply:
Yup will do.

Reply:
https://github.com/rapidsai/cuml/issues/6203